envi/archs/arm/disasm.py

```diff
@@ -2099,11 +2150,11 @@ def getOperAddr(self, op, emu=None):
 
         
         if (self.pubwl & 0x12) == 0x12:    # pre-indexed
-            if emu._forrealz: emu.setRegister( self.base_reg, addr)
+            if (emu != None) and (emu._forrealz): emu.setRegister( self.base_reg, addr)
```

Comment:
good catch.  if reg is PC, we already know it so we continue...  however, iinm, the spec says that behavior is undefined if reg == PC and Indexing.